datastore/datastore.go

```diff
@@ -38,13 +58,14 @@ const (
     }
 }
 `
-	delete_datastore_doc_query = `
-{
+	get_datastore_doc_query = `
+{"sort": {"timestamp": {"order": "desc"}},
+ "size": 1,
     "query": {
         "bool": {
             "must": [
                 {
-                    "prefix": {
+                    "match": {
```

Comment:
wasn't this using prefix because other queries rely on this also picking up ids that are in a format of `<id>_download`?